k8s/models/node.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+from __future__ import absolute_import
+
+import six
+
+from .common import ObjectMeta
+from ..base import Model
+from ..fields import Field, OnceField, ListField
+
+
+class NodeSpec(Model):
+    externalID = OnceField(six.text_type)
```

Comment:
This is tricky. In 1.8, this field was deprecated, but it will be removed in 1.13. I guess we will keep it for now.

Reading the documentation, I think this might be a `ReadOnlyField`, instead of a `OnceField`. At least in practice, since normal clients can't create a `Node` and set the value of the field.